export const GET = async () => {

        try { } catch (error) {

                console.error("couldn't get the user exams", error)

        }



}
